thunder/executors/nvfuserex_impl.py

```diff
@@ -383,10 +383,12 @@ class FusionDefinitionWrapper:
     cache_info: None | Callable = None
     cache_clear: None | Callable = None
     last_used: None | FusionDefinition = None
+    last_inputs_meta: None | Sequence[tuple] = None
 
     def __call__(self, *args):
         fd = self.get_fd(to_descriptors(args))
         self.last_used = fd
+        self.last_inputs_meta = [(i.size(), i.stride(), i.dtype, i.device) for i in args if isinstance(i, torch.Tensor)]
```

Comment:
The question @kevinstephano and I had was whether this is perf sensitive.

Reply:
That is also what I was thinking and the main reason why I was trying to save as little info as possible. However, maybe an idea would be to enable this behavior only when the practitioner is using a debug transform. Let me see what it's possible